core/src/main/scala/chisel3/internal/package.scala

```diff
@@ -152,4 +152,46 @@ package object internal {
       (zz, result.reverse)
     }
   }
+
+  /** The list of reserved keywords in FIRRTL which result in incorrect parsing when used in any statement outside
+    * of their correct use cases.
+    * This is vital for type aliases, which can in principle be named after these keywords but results
+    * in problematic and incorrect FIRRTL.
+    * Some words, like 'stop' and 'force', do not result in parser errors and consequently do not show up here.
+    */
```

Comment:
If we're excluding some FIRRTL keywords then I don't think it is correct to call this `firrtlKeywords`. If this really is just for legalizing type alias names, maybe just giving it name that describes the use case is better, eg. `illegalTypeAliasNames` or something.

Reply:
@dtzSiFive @seldridge do either of you know why some keywords would be problematic but others (`stop` and `force`) are not? @jared-barocsi It would be best to ensure we know why various keywords are legal alias identifiers and others are not.